src/attachment/index.ts

```diff
@@ -0,0 +1,351 @@
+              .id="${this._GUID}-input"
+              aria-required="${this.requiredIcon}"
+              aria-invalid="${this.error}"
+              aria-describedby="${this._GUID}-error"
+              @change="${this._handleChangeFiles}"></input>
+            </div>
+          </div>
+        </div>
+        <kuc-base-error
+          ?hidden="${!this.error}"
+          .text="${this.error}"
+          .guid="${this._GUID}"
+        ></kuc-base-error>
+      </div>
+    `;
+    }
+
+    private _getAttachmentItemTemplete(item: FileItem, index: number) {
+      return html`
+        <li class="kuc-attachment__group__files__display-area__item">
+          <div
+            title="${item.name || ""}"
+            class="kuc-attachment__group__files__display-area__item__name"
+          >
+            ${item.name || ""}
+          </div>
+          <div
+            class="kuc-attachment__group__files__display-area__item__remove-button__container"
+            ?hidden="${this.disabled}"
+          >
+            <button
+              class="kuc-attachment__group__files__display-area__item__remove-button__container__button"
+              aria-label="Cancel File"
+              data-file-index="${index}"
+              @click="${this._handleClickFileRemove}"
+              tabindex="0"
+            >
+              ${this._getRemoveButtonIcon()}
+            </button>
+          </div>
+          <span class="kuc-attachment__group__files__display-area__item__size">
+            ${this._getFileSize(item.size)}
+          </span>
+        </li>
+      `;
+    }
+    private _getRemoveButtonIcon() {
+      return html`<svg
+        xmlns="http://www.w3.org/2000/svg"
+        width="14"
+        height="14"
+        viewBox="0 0 14 14"
+        fill="none"
+      >
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
+          d="M5.93933 7.00001L1.96966 3.03034L1.43933 2.50001L2.49999 1.43935L3.03032 1.96968L6.99999 5.93935L10.9697 1.96968L11.5 1.43935L12.5607 2.50001L12.0303 3.03034L8.06065 7.00001L12.0303 10.9697L12.5607 11.5L11.5 12.5607L10.9697 12.0303L6.99999 8.06067L3.03032 12.0303L2.49999 12.5607L1.43933 11.5L1.96966 10.9697L5.93933 7.00001Z"
+          fill="#a8a8a8"
+        />
+      </svg>`;
+    }
+    private _getLanguage() {
+      const langs = ["en", "ja", "zh"];
+      if (langs.indexOf(this.language) !== -1) return this.language;
+
+      if (langs.indexOf(document.documentElement.lang) !== -1)
+        return document.documentElement.lang;
+
+      return "en";
+    }
+
+    private _getLocale() {
+      const language = this._getLanguage();
+      switch (language) {
+        case "en":
+          return en;
+        case "zh":
+          return zh;
+        case "ja":
+          return ja;
+        default:
+          return en;
+      }
+    }
+
+    private _handleClickFileRemove(event: any) {
```

Comment:
Change any to PointerEvent:
private _handleClickFileRemove(event: PointerEvent) {
      const removeButtonEl = event.currentTarget as HTMLButtonElement;
      const index = parseInt(
        removeButtonEl.getAttribute("data-file-index")!,
        10
      );